package.json

```diff
@@ -38,6 +39,7 @@
     "eslint-plugin-promise": "^6.0.0",
     "git-log-parser": "1.2.0",
     "husky": "^7.0.4",
+    "pnpm": "^6.32.8",
```

Comment:
Is this necessary? I think we can expect it to be available globally

Reply:
Yeah these can definitely be dropped, thanks for the catch